xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-processing/xwiki-platform-image-processing-plugin/src/test/java/com/xpn/xwiki/plugin/image/ImagePluginTest.java

```diff
@@ -183,10 +183,10 @@ void cacheOfScaledAttachment() throws Exception
 
     @ParameterizedTest
     @CsvSource({
-        "100, 500, false, 100, 300",
-        "500, 100, false, 400, 100",
+        "100, 500, false, 60, 300",
+        "500, 100, false, 400, 80",
+        "800, 600, false, 400, 300",
```

Comment:
Maybe it would also be interesting to add another case where both dimensions are larger than the input but don't match the input aspect ratio?

Reply:
done